src/xmipp/libraries/reconstruction/volumeset_align.cpp

```diff
@@ -133,13 +131,19 @@ void ProgVolumeSetAlign::computeFitness(){
 	const char * Volume2 = fnVolume2.c_str();
 
 	runSystem("xmipp_volume_align",formatString("--i1 %s --i2 %s --frm %f %d %d %d --store %s -v 0 ",
-			Volume1,Volume2,global_volumeset_align_prog->frm_freq, global_volumeset_align_prog->frm_shift, global_volumeset_align_prog->tilt0, global_volumeset_align_prog->tiltF, shifts_angles));
+			Volume1,Volume2,this->frm_freq, this->frm_shift, this->tilt0, this->tiltF, shifts_angles));
 	//The first 6 parameters are angles and shifts, and the 7th is the fitness value
 	fnAnglesAndShifts = fopen(shifts_angles, "r");
 	for (int i = 0; i < 6; i++){
-	       fscanf(fnAnglesAndShifts, "%f,", &global_volumeset_align_prog->Matrix_Angles_Shifts[i]);
+	       if(fscanf(fnAnglesAndShifts, "%f,", &this->Matrix_Angles_Shifts[i])!=1){
```

Comment:
Please store result of the call to some temporal variable, and then check it.
Use [REPORT_ERROR](https://github.com/I2PC/xmippCore/blob/cf86dd147051c3523c965d89920b6c51f69bcf23/core/xmipp_error.h#L238)